pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,117 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+
+from dataclasses import dataclass
+from pipeline_dp.aggregate_params import NoiseKind
+
+
+@dataclass
+class MeanVarParams:
+	"""Manages the parameters used for computing the mean/variance."""
+	eps: float
+	delta: float
+	low: float
+	high: float
+	sensitivity: float  # L1 for Laplace, L2 for Gaussian
```

Comment:
I thought again, and it's probably better to replace sensitivity with

**max_partitions_contributed**
**max_contributions_per_partition**

and then to compute sensitivity in this file using formulas from the design doc
https://docs.google.com/document/d/1cvfAU4XwglRgLc7XP9qCf5a6iZRajqVVsH7s5URY-UU/edit#bookmark=id.vpk6kpu64m5u

In that formula:
s0 is max_partitions_contributed
s_infinity is max_contributions_per_partition